searchbase/example/basic/web/main.dart

```diff
@@ -91,7 +91,7 @@ void main() {
   resultWidget.triggerDefaultQuery();
 
   resultWidget.subscribeToStateChanges((change) {
```

Comment:
The `change` property could be a map of `KeysToSubscribe` and `Changes`. You would have to make this change in the definition of subscription function in searchcontroller.